openshift_resources/backup.py

```diff
@@ -0,0 +1,13 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class Backup(NamespacedResource):
+
+    api_group = "oadp.openshift.io"
+
+    def __init__(self,
+                 hooks,
+                 namespaces_to_include,
```

Comment:
maybe add all of this params as kwargs with default value empty? as they are all optional...

Reply:
yes, thanks,  there will be init course
and yes, let's use xargs :)